podcasts.xcodeproj/project.pbxproj

```diff
@@ -13031,8 +13028,7 @@
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = au.com.shiftyjelly.podcasts.Clip;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "";
-				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "Pocket Casts App Clip Development";
+				PROVISIONING_PROFILE_SPECIFIER = "Pocket Casts App Clip Development";
```

Comment:
No need to specify this for iPhoneOS separately given it's the only one we have for this target.